src/logic/safe/utils/maxPriorityFeePerGas.ts

```diff
@@ -0,0 +1,31 @@
+import { hexToNumber } from 'web3-utils'
+
+import { getWeb3ReadOnly } from 'src/logic/wallets/getWeb3'
+
+export const clampMaxPrioFeePerGas = (maxPriorityFeePerGas: number, maxFeePerGas: number): number => {
+  if (maxPriorityFeePerGas > maxFeePerGas) {
+    const MIN_MAX_PRIO_FEE = 0.5
+    const MAX_MAX_PRIO_FEE = maxFeePerGas - 1
+
+    return Math.max(MIN_MAX_PRIO_FEE, MAX_MAX_PRIO_FEE)
```

Comment:
This can still result in in maxFee less than maxPrioFee, if maxFee is less than 0.5.

Reply:
This seems to be working this morning fine without round, but yesterday it was throwing. I guess it needs to be thoroughly tested.

Reply:
I mean, if e.g. `maxFee = 0.4, maxPrioFee = 0.5`, the tx wouldn't go through.